src/System.Windows.Forms.Primitives/src/Interop/Shell32/Interop.IDragSourceHelper2.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.ComTypes;
+
+internal static partial class Interop
+{
+    internal static partial class Shell32
+    {
+        [ComImport]
+        [Guid("DE5BF786-477A-11D2-839D-00C04FD918D0")]
+        [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+        public interface IDragSourceHelper2
+        {
+            HRESULT InitializeFromBitmap(
+                SHDRAGIMAGE pshdi,
+                IDataObject dataObject);
+
+            HRESULT InitializeFromWindow(
+                IntPtr hwnd,
+                Point ppt,
```

Comment:
```suggestion
                in Point ppt,
```

https://docs.microsoft.com/en-us/windows/win32/api/shobjidl_core/nf-shobjidl_core-idragsourcehelper-initializefromwindow

Reply:
The docs indicate that the 2nd and 3rd args are pointers, since `IDataObject` is a ref type, it's passed by ref. But shouldn't it be `Point* ppt`?

Reply:
I guess `in` is likely achieving the same, https://devblogs.microsoft.com/premier-developer/the-in-modifier-and-the-readonly-structs-in-c/. Though we generally strive to align with the native definitions where possible.

Reply:
`LPDATAOBJECT` in C++ is `IDataObject` in C# in order to add `ref` we need at least `LPLPDATAOBJECT` in C++

Reply:
I use `in` here, because other option to use `ref` and because `[in] Point*` I belive it's better capture that `[in]` modifier in C# which is possible nowadays.